exercises/dominoes/dominoes.exs

```diff
@@ -0,0 +1,12 @@
+defmodule Dominoes do
+
+  @doc """
+  chain?/1 takes a list of integer and returns boolean indicating if chain is possible
+  number to 1 when following the rules:
+    - if number is odd, multiply with 3 and add 1
+    - if number is even, divide by 2
+  """
+  @spec chain?(dominoes :: list) :: boolean
```

Comment:
`@type domino :: [1..6, 1..6]` is unfortunately not valid syntax (seems like one cannot specify the amount of members a list has. `@type domino :: {1..6, 1..6}` seems valid though. 
I'll have to change the code to use tuples instead of lists, is that OK?